landscape.yml

```diff
@@ -342,6 +342,14 @@ landscape:
             logo: salt-stack.svg
             twitter: https://twitter.com/Salt_Project_OS
             crunchbase: https://www.crunchbase.com/organization/saltstack
+          - item:
+            name: Sedai
+            description: >-
+            Sedai optimizes & implements configurations for containerized and serverless applications autonomously based on ML-based analysis of production behavior
```

Comment:
Please make sure to indent the description correctly - 2 spaces. You can check others with the description as examples.